pkg/synthetictests/resource_growth.go

```diff
@@ -123,3 +109,42 @@ func comparePostUpgradeResourceCountFromMetrics(testName, resource string) []*ju
 	}
 
 }
+
+func getAndCheckClusterVersion(testName string) (*v1.ClusterVersion, []*junitapi.JUnitTestCase) {
```

Comment:
Could we pass in the oc client rather than creating another one.

Reply:
yes

Reply:
This was missed, I don't fully know how heavy weight the client create is, probably not much but just to be safe it would be good to pass it through. Too often there's some obscure mechanism out there, maybe assuming we use the same client throughout a test or something.

Reply:
dang it. my bad. just updated and tested again with this change. thanks for catching it the 2nd time.